src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -48,13 +49,8 @@ private static RuntimeType GetTypeFromHandleSlow(IntPtr handle)
         [MethodImpl(MethodImplOptions.InternalCall)]
         private static extern RuntimeType? GetTypeFromHandleIfExists(IntPtr handle);
 
-        private static RuntimeType? GetTypeFromHandle(IntPtr handle)
+        private static RuntimeType GetRuntimeTypeFromHandle(IntPtr handle)
         {
-            if (handle == IntPtr.Zero)
-            {
-                return null;
-            }
-
             return GetTypeFromHandleIfExists(handle) ?? GetTypeFromHandleSlow(handle);
```

Comment:
Rename `GetTypeFromHandleIfExists` -> `GetRuntimeTypeFromHandleIfExists`, `GetTypeFromHandleSlow` -> `GetRuntimeTypeFromHandleSlow` to match `GetTypeFromHandle` -> `GetRuntimeTypeFromHandle` rename?